packages/clarity-js/src/layout/region.ts

```diff
@@ -44,13 +44,19 @@ export function exists(node: Node): boolean {
     return regionMap && regionMap.has(node);
 }
 
-export function track(id: number): void {
-    // Do not send another entry if we are already sending it
+export function track(id: number, event: Event): void {
     let node = dom.getNode(id);
-    if (updates.indexOf(id) <= 0 && regionMap.has(node)) {
-        regions[id] = id in regions ? regions[id] : { id, visible: BooleanFlag.False, region: regionMap.get(node) };
-        data.push(regions[id]);
-    }
+    let d = id in regions ? regions[id] : { id, state: Interaction.Rendered, name: regionMap.get(node) };
```

Comment:
Nit. What is d an abbrevation for. Can you use a slighlty more meaningful name?

Reply:
Changed it to data.